components/knative-function-controller/README.md

```diff
@@ -18,13 +18,39 @@ This Knative-based serverless implementation defines and handles the Function Cu
 ### Run locally
 Follow these steps to run the Knative Function controller locally:
 
-1. Modify the `config/config.yaml` file to include your base64-encoded `gcr.io` or `docker.io` credentials. 
-2. Update the Docker registry value to your `docker.io` username.
+1. Create a ServiceAccount to enable knativeBuild docker builds
 
-3. Apply the configuration:
+Currently you cannot run builds in every existing namespace of your cluster. ServiceAccounts with linked docker 
```

Comment:
```suggestion
Currently, you cannot run builds for every Namespace existing in your cluster. 
```

Reply:
the build cannot run if it is created IN a certain namespace, as it requires a serviceaccount in that namespace

Reply:
Maybe we can combine these two and have something like this then:

`To run a build in a Namespace, create ServiceAccounts with linked Docker repository credentials for each Namespace that will be used for the knative-function controller.`